### PR TITLE
Installer for Windows, run-web

### PR DESCRIPTION
**Pull request by My12123**

### CODE REVIEW

**Comment by nagadomi** [comment]
>update.sh

this completely unnecessary

>run-web.bat

This is too specific to your environment. Users cannot use this without modification.
However, there is a similar file in https://github.com/nagadomi/nunif/tree/master/waifu2x/web/appendix .
So I could put it there as an example.

>Installer for Windows.bat

This is difficult to decide how to handle at the moment.
It would be nice to have for Windows users, but there are many options, such as pip and conda, pytorch version (CUDA version, or CPU), etc.

**Comment by nagadomi** [comment]
- delete update.sh
- rename `run-web.bat` to `Run waifu2x Web.bat` and remove `--tile-size` option(This may cause errors on low memory GPU)

then I will approve this

File: Installer for Windows CPU.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Install python 3.10
```
Comment (nagadomi) [inline Installer for Windows CPU.bat:1]: `#` is not comment syntax for windows bat file.
Instead, use rem command
`@rem Install python 3.10`

Comment (My12123) [reply Installer for Windows CPU.bat]: Fixed

**Comment by nagadomi** [comment]
No need to change the directory for venv CUDA/CPU.
Use `venv`. It is already listed in .gitignore.

File: Run waifu2x Web CPU.bat
Comment (My12123) [inline Run waifu2x Web CPU.bat:1]: The old code led to an error due to -g 0 by default
```python
M:\nunif>call .\venv_nunif_CPU\Scripts\activate
Traceback (most recent call last):
  File "M:\WPy64-310110\python-3.10.11.amd64\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "M:\WPy64-310110\python-3.10.11.amd64\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "M:\nunif\waifu2x\web\__main__.py", line 9, in <module>
    from .server import main
  File "M:\nunif\waifu2x\web\server.py", line 154, in <module>
    command_args, config, art_ctx, art_scan_ctx, photo_ctx, cache, cache_gc = setup()
  File "M:\nunif\waifu2x\web\server.py", line 131, in setup
    art_ctx = Waifu2x(model_dir=args.art_model_dir, gpus=args.gpu)
  File "M:\nunif\waifu2x\utils.py", line 23, in __init__
    self.device = create_device(gpus)
  File "M:\nunif\nunif\device.py", line 21, in create_device
    return torch.device(create_device_name(device_id))
  File "M:\nunif\nunif\device.py", line 16, in create_device_name
    raise ValueError("No cuda/mps available. Use `--gpu -1` for CPU.")
ValueError: No cuda/mps available. Use `--gpu -1` for CPU.
Для продолжения нажмите любую клавишу . . .
```

**Comment by nagadomi** [comment]
Also I plan to fix the default --gpu option for CPU only env, so there is no need to separate Run waifu2x Web.bat.

**Comment by nagadomi** [comment]
No, auto detect
```
if torch.cuda.is_available() or torch.backends.mps.is_available():
    default_gpu = 0                                                
else:
    default_gpu = -1
```

**Comment by nagadomi** [comment]
Thank you. I will make the above fix and update INSTALL-windows.md later.

**Pull request merged**